internal_user/templates/init_fund_deposit.html

```diff
@@ -0,0 +1,51 @@
+{% extends "base.html" %}
+{% block javascript %}
+{% endblock %}
+{% block content %}
+
+<div>
+    <div>
+        <h2>Initiate fund deposit</h2><br><br>
+    </div>
+    <div class="form-group">
+        <form action='./searchCustomer' method="post">
+            <label>Search Customer:</label>
+            {% csrf_token %}
+            <input type="text" id="customerSearchString" class="form-control" name="customerSearchString" value={{customerSearchString}}><br>
+            <input type="hidden" name="context_page" value="deposit">
+            <input type="submit" value="Search" class="btn btn-primary float-right app-submit"><br><br>
+        </form>
+    </div>
```

Comment:
I agree that we need to keep all features as simple as possible. We anyways have to implement customer search functionality for View customer feature. Therefore we are just reusing search customer function in other features. This will not cause us any extra overhead